manifests/03-rbac-role-ns-openshift-config.yaml

```diff
@@ -8,6 +8,7 @@ rules:
       - ""
     resources:
       - configmaps
+      - secrets
```

Comment:
Too bad we have to add this, I liked that we didn't read secrets from here.

Reply:
Not that we ARE reading them all, just that now we can. but we are only doing a GET on the one that matters to us.

Reply:
Other solution would be to have the secret name as API, then we could specify only that single one... but not sure if thats worth it.